Containerfile

```diff
@@ -1,19 +1,15 @@
-FROM registry.fedoraproject.org/fedora:39 AS builder
-RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/bib
+FROM registry.access.redhat.com/ubi9/ubi:latest AS builder
+RUN dnf install -y git-core golang && mkdir -p /build/bib
 COPY bib/go.mod bib/go.sum /build/bib
 RUN cd /build/bib && go mod download
 COPY build.sh /build
 COPY bib /build/bib
 WORKDIR /build
 RUN ./build.sh
 
-FROM registry.fedoraproject.org/fedora:39
-# Install newer osbuild to fix the loop bug, see
-# - https://github.com/osbuild/bootc-image-builder/issues/7
-# - https://github.com/osbuild/bootc-image-builder/issues/9
-# - https://github.com/osbuild/osbuild/pull/1468
-COPY ./group_osbuild-osbuild-fedora-39.repo /etc/yum.repos.d/
-RUN dnf install -y osbuild osbuild-ostree osbuild-depsolve-dnf && dnf clean all
+FROM quay.io/centos/centos:stream9
+# FROM registry.access.redhat.com/ubi9/ubi:latest
+RUN dnf install -y osbuild osbuild-ostree  && dnf clean all
```

Comment:
Pardon my ignorance on this - we will want to install more tools in the future into the bib container:
- qemu-img so that we can convert the raw -> cqow2
- qemu-user so that we can cross build images when host arch != target arch, the idea is that the bootc image must be available for both arches and then the buildroot is setup with the host arch and we build the target image but some (small) tasks will happen inside the target arch tree via chroot and for that qemu-user and a binfmt_misc handler is needed to handle this transparently

Will this be an issue when we move here?

Reply:
Yes, moving the buildroot requirements here makes sense to me.  However, that is orthogonal to this PR because we still end up running ELN today regardless, right?